dnf5daemon-server/package.hpp

```diff
@@ -72,4 +72,10 @@ enum class PackageAttribute {
 dnfdaemon::KeyValueMap package_to_map(
     const libdnf5::rpm::Package & libdnf_package, const std::vector<std::string> & attributes);
 
+/// Convert given libdnf_package to a JSON string using requested attributes.
+/// @param libdnf_package Package to convert to JSON
+/// @param attributes A list of attributes of ligdnf_package that are included in JSON
+/// @return JSON String with the package represented as key:value dictionary.
+std::string package_to_json(const libdnf5::rpm::Package & libdnf_package, const std::vector<std::string> & attributes);
+
```

Comment:
I am not sure whether it is important, but `package.hpp` and `cpp` does not use any namespaces.

Reply:
I know I've already started moving stuff into namespace in this PR and if you want me to, I can continue with `package.[ch]pp`. But personally I'd prefer to create separate issue for move to namespace and resolve it in separate PR. Are you OK with this?

Reply:
Created an issue for this refactoring: https://github.com/rpm-software-management/dnf5/issues/1389